frontend/components/TeamsDropdown/TeamsDropdown.tsx

```diff
@@ -121,12 +127,13 @@ const TeamsDropdown = ({
       padding: "8px 0",
       backgroundColor: "initial",
       border: 0,
+      borderRadius: "4px",
       boxShadow: "none",
       cursor: "pointer",
       "&:hover": {
         boxShadow: "none",
         ".team-dropdown__single-value": {
-          color: COLORS["core-vibrant-blue-over"],
+          color: COLORS["core-fleet-blue"],
```

Comment:
Just so I'm clear, `ui-vibrant-blue-10` is for focused and `ui-vibrant-blue-25` is for active (and no special treatment for selected)? And we want on-hover here in particular to be `core-fleet-blue` (I'm guessing we already have that hover set elsewhere?), and foreground old-style dropdown elements to be `$core-vibrant-blue-down` when activated? And I'm guessing none of these four colors are equivalent?

Sorry for basically requesting a quick rehash of the thread on this that I read earlier (don't have access to the associated Figma)...I'm pretty sure this is okay here, but want to understand what I'm looking at, and an answer here may be useful for QA.

Reply:
Okay, got access to Figma. Guessing the color I'm seeing in Figma is `ui-vibrant-blue-10`, with `ui-vibrant-blue-25` showing up for the split second a menu item is clocked?

Reply:
Okay, yeah, I think I'm understanding things. An explanation of `core-fleet-blue` and `$core-vibrant-blue-down` would be helpful, but from what I can tell we're good to go here (after removing the unused import).